src/inverseRobot.js

```diff
@@ -24,7 +24,18 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let result = {};
+
+  for (const key in robot) {
+    if (result[robot[key]]) {
+      result = null;
+      break;
```

Comment:
why do you need this `break`?

Reply:
when I find a repeated object, I don't need to look for it anymore